go/eks/cluster-graviton/README.md

```diff
@@ -0,0 +1,27 @@
+# Graviton EKS Cluster
+
+This is an example of how to provision EKS cluster with AWS Graviton nodes with CDK.
+
+**NOTICE**: Go support is still in Developer Preview. This implies that APIs may
```

Comment:
It would be good to use the official stability banner from https://github.com/aws-samples/aws-cdk-examples/blob/master/CONTRIBUTING.md

Reply:
Thanks, will add it.